x-pack/test/functional/services/ml/settings_filter_list.ts

```diff
@@ -17,7 +17,7 @@ export function MachineLearningSettingsFilterListProvider({ getService }: FtrPro
       const $ = await table.parseDomContent();
       const rows = [];
 
-      for (const tr of $.findTestSubjects('~mlCalendarListRow').toArray()) {
+      for (const tr of $.findTestSubjects('~mlFilterListsTable').toArray()) {
```

Comment:
This should also be updated to `mlFilterListsRow`.

Reply:
Updated here https://github.com/elastic/kibana/pull/76735/commits/1fa702690b0cd8a4a4b65498cb2fad5d6d46f5df